### DISCUSSION TITLE
Separate rtsp stream for Audio and Video

### DISCUSSION THREAD

**Discussion by nokmanok** [blakeblackshear/frigate#8847] [Ask A Question] [answered]
I have a camera with rtsp video stream:
rtsp://some_name:some_pass@ip_add:port/videoMain
and an audio rtsp stream:
rtsp://some_name:some_pass@ip_add:port/audio

Right now I have set recordings for the video stream but no luck pairing it with the audio stream.
Need help with this one. Thanks

**Comment by NickM-27** [comment, marked as answer]
You can add both to go2rtc stream and then use the go2rtc restream as the source.

**Reply by nokmanok** [reply]
It works! Terrible sorry if I posted a noob question. First time setting up nvr and frigate

go2rtc:
  streams:
    Parking:
      - rtsp://parking:{FRIGATE_RTSP_PASSWORD}@IP_ADD:PORT/videoMain
      - rtsp://parking:{FRIGATE_RTSP_PASSWORD}@IP_ADD:PORT/audio
      - "ffmpeg:Parking#audio=aac"
